src/v2/Apps/Artist/ArtistApp.tsx

```diff
@@ -45,8 +45,26 @@ const ArtistApp: React.FC<ArtistAppProps> = ({ artist, children, match }) => {
   }
 
   const showOverviewTab = hasOverviewContent(artist)
-  const showArtworksTab = artist?.statuses?.artworks
-  const showAuctionResultsTab = artist?.statuses?.auctionLots
+  const showAuctionResultsTab =
+    artist?.statuses?.auctionLots || artist?.auctionResultsStatus?.edges?.length
```

Comment:
(Discussed a little [here](https://artsy.slack.com/archives/C01ADJNCS5D/p1643376118098399?thread_ts=1643351582.972189&cid=C01ADJNCS5D).) I think we should fix the API's `display_auction_link` property, since its whole purpose is to help with this logic. It also respects the `Artist#hide_auction_link` property which presumably is important.

When I tested one of the artists referred to in [the ticket](https://artsyproduct.atlassian.net/browse/GRO-682), it already had a positive count and a visible tab. The other did not, which is possible when the artist's record has never been "touched" via follows, artworks, sales, or shows. We should probably refresh these counts periodically, rather than depending on those events. Even better, Diffusion could publish an event when there are new [published] auction results for an artist, and Gravity could subscribe to it and refresh then.